src/representation_theory/basic.lean

```diff
@@ -374,6 +374,13 @@ def dual : representation k G (module.dual k V) :=
 @[simp]
 lemma dual_apply (g : G) : (dual ρV) g = module.dual.transpose (ρV g⁻¹) := rfl
 
+/--
+Given `k`-modules `V, W`, there is an isomorphism `φ : V^* ⊗ W → Hom_k(V, W)`
+(implemented by `dual_tensor_hom`).
```

Comment:
`dual_tensor_hom` does not implement the isomorphism, it is only the homomorphism.

Reply:
good point, fixed

Reply:
We might want to mention `dual_tensor_hom_equiv`, which is the isomorphism version.

Reply:
Hm, yes, maybe it would be a good exercise for me to put together a short lemma proving that the two spaces are isomorphic as G-modules. Called something like `dual_tensor_hom_equiv_comm` ?

Reply:
Never mind, the isomorphism version is dealt with in `fdRep.lean`, so I won't reimplement that. I'm not sure it makes sense to mention it here, what do you think?

Reply:
You're right, I think it's ok not to mention the isomorphism version here.